diff --git a/bin/compile b/bin/compile
index d92bd73..412208b 100755
--- bin/compile
+++ bin/compile
@@ -150,6 +150,9 @@ fi
 LIBPCRE_URL="http://iweb.dl.sourceforge.net/project/pcre/pcre/8.35/pcre-8.35.tar.gz"
 install_prebuilt_library "libpcre" "8.35" $LIBPCRE_URL "pcre-8.35" "--enable-silent-rules --enable-static=no --disable-cpp"
 
+LIBICU_URL="http://download.icu-project.org/files/icu4c/52.1/icu4c-52_1-src.tgz"
+install_prebuilt_library "libicu" "52.1" $LIBICU_URL "icu/source"
+
 LIBGMP_URL="https://gmplib.org/download/gmp/gmp-6.0.0a.tar.bz2"
 install_prebuilt_library "libgmp" "6.0.0a" $LIBGMP_URL "gmp-6.0.0"
